driver/src/main/scala/core/nodeset.scala

```diff
@@ -1,19 +1,34 @@
 package reactivemongo.core.nodeset
 
-import java.util.concurrent.{ Executor, Executors, TimeUnit }
+import java.util.concurrent.{ TimeUnit, Executor, Executors }
 
 import scala.collection.generic.CanBuildFrom
 import scala.collection.immutable.Set
+
 import shaded.netty.util.HashedWheelTimer
 import shaded.netty.buffer.HeapChannelBufferFactory
 import shaded.netty.channel.socket.nio.NioClientSocketChannelFactory
-import shaded.netty.channel.{ Channel, ChannelFuture, ChannelPipeline, Channels }
+import shaded.netty.channel.{
```

Comment:
I think that's me correcting them back to what it should be, see overall changes between my branch and master